packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -91,7 +99,7 @@ function FeedNav(): ReactElement {
         </TabContainer>
 
         {isMobile && sortingEnabled && isSortableFeed && (
-          <div className="sticky flex h-11 w-20 -translate-y-12 translate-x-[calc(100vw-100%)] items-center justify-end bg-gradient-to-r from-transparent via-background-default via-40% to-background-default pr-4">
+          <StickyNavIconWrapper className="translate-x-[calc(100vw-100%)]">
```

Comment:
Can't we do the className with `translate-x-[calc(100vw-100%)] tablet:translate-x-[calc(100vw-180%)]`

Reply:
It's 2 different icons, this is sorting, the one we're fixing is notifications which show on different places.

Reply:
But they both the same component? I mean the StickyNavIconWrapper

Reply:
It's the wrapper but it should show on different places for different icons, not on different places for tablet/mobile so this way we can decide where to show the wrapper +component based on the component